CHANGELOG.md

```diff
@@ -126,7 +126,7 @@ The following changes are recommendations and don't have to be done as we will s
 
 ### Other Breaking changes
 
-* The log importer in `misc/log-analytics` now supports Python 3 (3.5, 3.6 or 3.7), it will no longer run with Python 2. If you have any automated scripts that run the importer, you will have to change them to use the Python 3 executable instead.
+* The log importer in `misc/log-analytics` now supports Python 3 (3.5, 3.6, 3.7 or 3.8), it will no longer run with Python 2. If you have any automated scripts that run the importer, you will have to change them to use the Python 3 executable instead.
```

Comment:
fyi added a note to https://github.com/matomo-org/matomo/wiki/Matomo-4.0.0-release-notes

Can people maybe still checkout an earlier version from the log importer repository supporting Python 2? Could mention it maybe but should be enough to have this mentioned in the announcement as it's not really developer related.

Reply:
It should definitely be mentioned in the release notes with maybe a link to an FAQ explaining how to switch over.

I don't know of any breaking change that would cause the old script to not work with Matomo  4